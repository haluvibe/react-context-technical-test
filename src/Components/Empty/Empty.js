import React from 'react';

const Empty = () => <div>Your favourites list is empty</div>;

export default Empty;
