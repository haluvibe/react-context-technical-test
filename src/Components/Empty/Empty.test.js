import React from 'react';
import { shallow } from 'enzyme';
import Empty from './Empty';

describe('Empty', () => {
  it('should render correctly with no props', () => {
    const wrapper = shallow(<Empty />);

    expect(wrapper).toMatchSnapshot();
  });
});
