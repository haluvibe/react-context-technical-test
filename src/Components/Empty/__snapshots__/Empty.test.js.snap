// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty should render correctly with no props 1`] = `
<div>
  Your favourites list is empty
</div>
`;
